### PR TITLE
refact(yaml): Dump data to use the resized volume after csi volume resize

### PR DESCRIPTION
**Pull request by w3aman**
Signed-off-by: w3aman <aman.gupta@mayadata.io>

**What this PR does / why we need it**:

- This PR adds task in csi-volume resize test to dump dummy data after resizing the csi volume to verify that volume can be filled in the resized space.

### CODE REVIEW

File: experiments/functional/csi-volume-resize/test.yml [hunk 1]
```diff
@@ -79,6 +107,30 @@
...
+              shell: >
+                  kubectl exec -it "{{ app_pod.stdout }}" -n "{{ app_ns }}" 
+                  -- sh -c "cd {{ mount.stdout }} && dd if=/dev/urandom of=volume.txt bs=1024k count={{ value_num }}"
+              args:
+                  executable: /bin/bash
+              register: load
+              failed_when: "load.rc != 0"
+              when: "mount.stdout == '/busybox'"
+
+            - name: Dump some dummy data in the application mount point for using resized volume when application is percona
```
Comment (nsathyaseelan) [inline experiments/functional/csi-volume-resize/test.yml]: @w3aman why there is two same tasks for dumping the data

**Review by nsathyaseelan — CHANGES_REQUESTED** [review]

File: experiments/functional/csi-volume-resize/test.yml [hunk 2]
```diff
@@ -38,6 +38,34 @@
...
+              shell: >
+                kubectl get pod -n {{ app_ns }} -l {{ app_label }} --no-headers
+                -o custom-columns=:.metadata.name
+              args:
+                executable: /bin/bash
+              register: app_pod
+
+            - name: Obtain the mount path for the application
+              shell: > 
+                  kubectl get pods -n {{ app_ns }} -l {{ app_label }}
```
Comment (gprasath) [inline experiments/functional/csi-volume-resize/test.yml]: {{ app_pod.stdout }} can be used instead

File: experiments/functional/csi-volume-resize/test.yml (hunk 1 above)
Comment (w3aman) [reply experiments/functional/csi-volume-resize/test.yml]: for busybox and percona, we have diff mount points so for that i added two diff task seperated by when condition

Comment (nsathyaseelan) [reply experiments/functional/csi-volume-resize/test.yml]: In this tasks mount point is not hard coded, we were obtaining the mount point and then only we are using so only one task is enough i think

**Review by nsathyaseelan — CHANGES_REQUESTED** [review]

File: experiments/functional/csi-volume-resize/test.yml (hunk 1 above)
Comment (w3aman) [reply experiments/functional/csi-volume-resize/test.yml]: Correct, i will update

**Review by nsathyaseelan — APPROVED** [review]
lgtm

**Pull request merged**